lambda/src/main/resources/db/migration/V97__update_displayproperties.sql

```diff
@@ -0,0 +1,43 @@
+-- Update the displayProperties
+
+UPDATE "DisplayProperties"
+SET "Value" = 'Date of the Advisory Council approval.'
+WHERE "PropertyName" = 'FoiExemptionAsserted' AND "Attribute" = 'Description';
+
+UPDATE "DisplayProperties"
+SET "Value" = 'The date of the last meaningful change to this record.'
+WHERE "PropertyName" = 'ClosureStartDate' AND "Attribute" = 'Description';
+
+UPDATE "DisplayProperties"
+SET "Value" = 'This is the number of years the record is closed from the closure start date. If your file has multiple closure periods, enter the longest.'
+WHERE "PropertyName" = 'ClosurePeriod' AND "Attribute" = 'Description';
+
+UPDATE "DisplayProperties"
+SET "Value" = 'FOI exemption code'
+WHERE "PropertyName" = 'FoiExemptionCode' AND "Attribute" = 'Name';
+
+UPDATE "DisplayProperties"
+SET "Value" = 'Is the title sensitive for the public?'
+WHERE "PropertyName" = 'TitleClosed' AND "Attribute" = 'Name';
+
+UPDATE "DisplayProperties"
+SET "Value" = 'Is the description sensitive for the public?'
+WHERE "PropertyName" = 'DescriptionClosed' AND "Attribute" = 'Name';
+
+UPDATE "DisplayProperties"
+SET "Value" = 'Add alternative title without file extension'
+WHERE "PropertyName" = 'TitleAlternate' AND "Attribute" = 'Name';
+
+UPDATE "DisplayProperties"
+SET "Value" = 'Alternative description'
+WHERE "PropertyName" = 'DescriptionAlternate' AND "Attribute" = 'Name';
+
+UPDATE "DisplayProperties"
+SET "Value" = ''
+WHERE "PropertyName" = 'TitleClosed' AND "Attribute" = 'Description';
```

Comment:
I have set the value for the description for TitleClosed to blank because in the prototype, it doesn't have a description. The black text is just extra information according to Gareth. So I'm not sure what attribute this should be.

Reply:
@thanhz Can you add a note to that task TDR-2852 about adding this.